docker/nginx/conf.d/include/track-upload

```diff
@@ -5,14 +5,13 @@ log_by_lua_block {
 
     local skylink = ngx.header["Skynet-Skylink"]
     if skylink and ngx.status >= ngx.HTTP_OK and ngx.status < ngx.HTTP_SPECIAL_RESPONSE then
-        local http = require("socket.http")
-        local ok, statusCode, headers, statusText = http.request {
-            url = "http://accounts:3000/track/upload/" .. skylink,
+        local httpc = require("resty.http").new()
+        local res, err = httpc:request_uri("http://10.10.10.70:3000/track/upload/" .. skylink, {
```

Comment:
Is there a reason to use IPs instead of names? I see you switched from `accounts` to `10.10.10.70` - is that for easier find-and-replace type of task or is there a different reason? I find it a little harder to read (constantly checking docker-compose) but if that works better for you, that's OK.

Reply:
unfortunately `httpc:request_uri` doesn't have access to resolved aliases, I suspect that it operates in some kind of forked process or sth so passing aliases like `sia` or `account` doesn't work there

Reply:
Oh, good to know. It might be nice to leave a small comment explaining which service we're targeting? Definitely not a necessity, though.

Reply:
> Oh, good to know. It might be nice to leave a small comment explaining which service we're targeting? Definitely not a necessity, though.

added in https://github.com/SkynetLabs/skynet-webportal/pull/1075/commits/b339aeccc96478119470b6dd3f6d9093d40a253e